### PR TITLE
web-ui: Add logout

### PR DESCRIPTION
**Pull request by bhaskarmelkani**

### CODE REVIEW

File: web-ui/README.md [hunk 1]
```diff
@@ -68,6 +69,16 @@ DEBUG=false
 * **OAUTH_TOKENINFO_URL**: The url used to validate the access token and retrieve token information
 * **OAUTH_REFRESH_TOKEN_URL**: The url used to refresh the access token
 * **OAUTH_SCOPES**: Comma separated list of scopes that the user should grant to the app
+* **OAUTH_USERNAME_FIELD_PATH**: Path of the field that stores username in the retrived token information.
+Eg.
```
Comment (rbarilani) [inline web-ui/README.md]: I would suggest something like:

**OAUTH_USERNAME_PROPERTY**: Property that can be found in the ```/tokeninfo``` response representing the username of the connected user (eg. ```uid``` or ```user.uid``` if nested)

File: web-ui/src/server/env.js [hunk 2]
```diff
@@ -19,7 +19,8 @@ const defaults = {
   OAUTH_SCOPES:'',
   DEV_PORT: 8441,
   DEV_SSL_ENABLED: false,
-  DEBUG: false
+  DEBUG: false,
+  OAUTH_USERNAME_FIELD_PATH: 'cn'
```
Comment (rbarilani) [inline web-ui/src/server/env.js]: After change the property to  `OAUTH_USERNAME_PROPERTY` reorder the usage so is grouped with the others OAUTH_* properties.

File: web-ui/src/client/app/components/user-info-container.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+export default function UserInfoContainer(props){
+
+  const showUserInfo = window.env.OAUTH_ENABLED;
+
+  if(!showUserInfo){
+    return null;
+  }
+
+  const css = `
```
Comment (rbarilani) [inline web-ui/src/client/app/components/user-info-container.jsx]: Move the style to top level

File: web-ui/README.md (hunk 1 above)
Comment (bhaskarmelkani) [inline web-ui/README.md]: Done

File: web-ui/src/server/env.js (hunk 2 above)
Comment (bhaskarmelkani) [inline web-ui/src/server/env.js]: Done

File: web-ui/src/client/app/components/user-info-container.jsx (hunk 3 above)
Comment (bhaskarmelkani) [inline web-ui/src/client/app/components/user-info-container.jsx]: Done

**Comment by bhaskarmelkani** [comment]
👍

**Pull request merged**